internal/cli/atlas/clusters/add.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clusters
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type AddOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	name  string
+	store store.SampleDataAdder
+}
+
+func (opts *AddOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+var addTmpl = "Loading sample dataset into {{.ClusterName}}.\n"
+
+func (opts *AddOpts) Run() error {
+	r, err := opts.store.AddSampleData(opts.ConfigProjectID(), opts.name)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas cluster add <clusterName> --projectId projectId -o json
```

Comment:
this command reads really weird, there's no way to know whiteout reading docs that `add` will load a sample dataset
This doesn't tell me anything about what the command does
```
mongocli atlas cluster add <clusterName>
```

Probably 
```
mongocli atlas cluster loadSampleData
```
is a better command here